cas-server-webapp/src/main/resources/messages.properties

```diff
@@ -84,6 +84,7 @@ authenticationFailure.InvalidLoginLocationException=You cannot login from this w
 authenticationFailure.InvalidLoginTimeException=Your account is forbidden to login at this time.
 authenticationFailure.AccountNotFoundException=Invalid credentials.
 authenticationFailure.FailedLoginException=Invalid credentials.
+authenticationFailure.UnauthorizedServiceForPrincipalException=Service access denied due to missing privileges.
```

Comment:
This messages show up on the screen similar to "Invalid Credentials", on top of the login box.